.gitignore

```diff
@@ -13,5 +13,4 @@ _START_PACKAGE
 .vscode/*
 !.vscode/extensions.json
 .idea
-vetur.config.js
-nodelinter.config.json
+nodelinter.config.json
```

Comment:
Add and start tracking `vetur.config.js` to [prevent VSCode from reporting non-errors in `/editor-ui` imports](https://github.com/n8n-io/n8n/pull/1841/). Most of the team is on VSCode.